src/components/Camera/index.js

```diff
@@ -56,7 +69,23 @@ const CameraPure = ({
           {...{ onUserMedia, ref: webcamRef, onFailure }}
         />
       </div>
-      <div id='cameraViewAriaLabel' aria-label={translate('accessibility.camera_view')}></div>
+      <div
+        className={classNames(style.actions, {
+          [style.disabled]: !hasGrantedPermission || isCaptureDisabled
+        })}
+      >
+        {!hideCaptureButton &&
```

Comment:
I can see you are setting`hideCaptureButton` to true in the parent component. At which point do you set it back to false?

Reply:
Also couldn't we just use the `isCaptureDisabled` prop to achieve this?

Reply:
The `hideCaptureButton` doesn't get set back to false in the parent component since it's not dependent on any state change. Doc Auto Capture screen doesn't require a capture button and Video Capture screen seems to have its own start/stop recording button, handlers.

Reply:
I'm not too sure about this change though because the camera component is used by both video, selfie and document autocapture components. Video handles its own button in the video component so this button will only be used by selfie. I think it makes sense that Camera component handles the buttons but if we only do this for selfie it might generate some confusion.

Reply:
Yeah, makes sense. Will look into bringing the video's button(s) to be also handled within Camera component for consistency.